cluster-operator/src/main/java/io/strimzi/operator/cluster/model/Utils.java

```diff
@@ -81,7 +81,7 @@ public static String getJsonCorruptionBlame(DecodeException de, String config) {
                 return "Unexpected character - " + token;
             }
             if (token.contains("Failed to decode: Unexpected end-of-input")) {
-                return "JSON braces";
+                return "JSON bracing";
```

Comment:
This is concatenated with "corruptes JSON" later. It would not be correct, so I workarounded it.

Reply:
What would the final message be then?

Reply:
`throw new InvalidConfigMapException(key, " is corrupted");`

in this case: JSON bracing is corrupted